plugin_tests/source_mapnik_test.py

```diff
@@ -209,3 +209,75 @@ def testThumbnailFromGeotiffs(self):
         image2 = self.getBody(resp, text=False)
         self.assertEqual(image2[:len(common.PNGHeader)], common.PNGHeader)
         self.assertNotEqual(image, image2)
+
+    def testPixel(self):
+        file = self._uploadFile(os.path.join(
+            os.path.dirname(__file__), 'test_files', 'rgb_geotiff.tiff'))
+        itemId = str(file['itemId'])
+
+        # Test in pixel coordinates
+        resp = self.request(
+            path='/item/%s/tiles/pixel' % itemId, user=self.admin,
+            params={'left': 212, 'top': 198})
+        self.assertStatusOk(resp)
+        self.assertEqual(resp.json, {
+            'r': 62, 'g': 65, 'b': 66, 'a': 255, 'bands': {'1': 62.0, '2': 65.0, '3': 66.0}})
+        resp = self.request(
+            path='/item/%s/tiles/pixel' % itemId, user=self.admin,
+            params={'left': 2120, 'top': 198})
+        self.assertStatusOk(resp)
+        self.assertEqual(resp.json, {})
+
+        # Test with a projection
+        resp = self.request(
+            path='/item/%s/tiles/pixel' % itemId, user=self.admin,
+            params={
```

Comment:
Can we add one more test with WGS 84 coordinates and see if we get the same results?
I did the conversion and it would be something like this:

```
params={
    'left': -117.97493778,
    'top': 33.8642302306,
    'projection': 'EPSG:4326',
    'units': 'projection'
}
```

Reply:
After checking this out second time I realized https://github.com/girder/large_image/blob/375c2c7e107ecc819d27313efd4bf6d000176c6f/server/tilesource/mapniksource.py#L97-L100. So I am happy.